cmd/utils.go

```diff
@@ -93,19 +94,19 @@ func path2BucketObject(s string) (bucket, prefix string) {
 }
 
 func getDefaultParityBlocks(drive int) int {
-	return drive / 2
+	return int(math.Floor(float64(drive) / 2))
```

Comment:
How it that different, except slower?

Reply:
> How it that different, except slower?

Currently we support an odd number of disks and a write quorum should be more than half. If we have 5 disks, 5/2 = 2.5, parity is always less than data, then we do math.Floor() for parity and math.Ceil() for data.

Reply:
@vadmeste 5/2 = 2 when you do integer division.

Reply:
> @vadmeste 5/2 = 2 when you do integer division.

yes correct, I thought you were asking about the whole purpose of the PR

Reply:
So just revert this change since the result will be the same.

Reply:
> So just revert this change since the result will be the same for reasonable values of drive.

Done.